api/types_test.go

```diff
@@ -0,0 +1,132 @@
+package api
+
+import (
+	"testing"
+)
+
+var (
+	OIDCNotEnabled     = false
```

Comment:
Can we move the package-level variables to the method they are used?

This will avoid polluting the package namespace.
In general variables should be declared with the minimum scope needed.